src/providers/export/tensorFlowRecords/tensorFlowBuilder.ts

```diff
@@ -0,0 +1,148 @@
+import { TFRecordsImageMessage, Features, Feature, FeatureList,
+    BytesList, Int64List, FloatList } from "./tensorFlowRecordsProtoBuf_pb";
+import { crc32c, maskCrc, getInt64Buffer, getInt32Buffer, textEncode } from "./tensorFlowHelpers";
+
+/**
+ * @name - TFRecords Builder Class
+ * @description - Create a TFRecords object
+ */
+export class TFRecordsBuilder {
+    /**
+     * @records - An Array of TFRecord Buffer created with releaseTFRecord()
+     * @description - Return a Buffer representation of a TFRecords object
+     */
+    public static releaseTFRecords(records: Buffer[]): Buffer {
```

Comment:
Same here - the name is a bit confusing.  Maybe rename to some like `joinBuffers` or joinRecords` ?